### PR TITLE
 prepare for v0.3.0

### PR DESCRIPTION
**Pull request by shrimalmadhur**

### CODE REVIEW

**Review by ian-shim — APPROVED** [review]
What calls srs_setup? Will that be a separate PR?

File: srs_setup.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Path: srs_setup.sh
+
+echo "Downloading srs resources"
+echo "Downloading g1 point. This could take upto 5 minutes"
+wget https://srs-mainnet.s3.amazonaws.com/kzg/g1.point --output-document=./resources/g1.point
+echo "Downloading g2 point. This could take upto 10 minutes"
+wget https://srs-mainnet.s3.amazonaws.com/kzg/g2.point --output-document=./resources/g2.point
+
+echo "validating hashes of g1 and g2 points This could take upto 5 minutes"
+cd resources && sha256sum -c srssha256sums.txt
```
Comment (ian-shim) [inline srs_setup.sh:11]: what happens when this checksum doesn't pass?

File: README.md [hunk 2]
```diff
@@ -49,6 +48,7 @@ In order to limit traffic from the EigenLabs hosted Disperser, please restrict y
 For users with private IP, you may need to perform port forwarding to open the dispersal ports. Refer to retrieval setup for more details.
 
 ### Opt-in into EigenDA
+This command also downloads the latest SRS points if they don't exist and can take around 20 minutes to complete for the firs time.
```
Comment (ian-shim) [inline README.md:51]: s/firs/first

File: run.sh [hunk 3]
```diff
@@ -9,6 +9,7 @@
 # To test that try running `docker run --rm --env-file .env busybox /bin/sh -c 'echo $NODE_ECDSA_KEY_PASSWORD'`
 # This will output password with single quote. Not sure why this happens.
 optIn() {
+  ./srs_setup.sh
```
Comment (ian-shim) [inline run.sh:12]: do we need to update the version in L20 and L34?

Comment (ian-shim) [inline run.sh:12]: should it shortcircuit if this script fails?

Comment (shrimalmadhur) [reply run.sh:12]: yea! that we need to update anyway in the release - I will add it now so we don't forget

Comment (shrimalmadhur) [reply run.sh:12]: yea I guess we should not opt-in if anything - downloading or validating SRS points fails

File: srs_setup.sh [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  echo "g2.point does not exist."
+  echo "Downloading g2 point. This could take upto 10 minutes"
+  wget https://srs-mainnet.s3.amazonaws.com/kzg/g2.point --output-document=./resources/g2.point
+  DOWNLOADED_FILE=true
+fi
+
+# Any time we download the file, validate hashes
+if [ "$DOWNLOADED_FILE" = true ]; then
+  echo "validating hashes of g1 and g2 points This could take upto 5 minutes"
+  if (cd resources && sha256sum -c srssha256sums.txt); then
```
Comment (ian-shim) [inline srs_setup.sh:23]: has this been tested?

**Review by ian-shim — APPROVED** [review]
lgtm as long as its tested

File: srs_setup.sh (hunk 4 above)
Comment (shrimalmadhur) [reply srs_setup.sh:23]: yes. have been testing this with correct and wrong sha.

File: .env.example [hunk 5]
```diff
@@ -10,12 +10,13 @@ NODE_PRIVATE_KEY=
...
 NODE_CACHE_ENCODED_BLOBS=true
 NODE_NUM_WORKERS=1
-NODE_SRS_ORDER=524288
 NODE_DISPERSAL_PORT=32005
 NODE_QUORUM_ID_LIST=0
 NODE_VERBOSE=true
 NODE_RETRIEVAL_PORT=32004
 NODE_TIMEOUT=20s
+NODE_SRS_ORDER=268435456
+NODE_SRS_LOAD=1048576
```
Comment (ian-shim) [inline .env.example:19]: should be 131072

Comment (shrimalmadhur) [reply .env.example]: updated

File: srs_setup.sh [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+if ! [ -f ./resources/g2.point ]; then
+  echo "g2.point does not exist."
+  echo "Downloading g2 point. This could take upto 10 minutes"
+  wget https://srs-mainnet.s3.amazonaws.com/kzg/g2.point --output-document=./resources/g2.point
+  DOWNLOADED_FILE=true
+fi
+
+# Any time we download the file, validate hashes
+if [ "$DOWNLOADED_FILE" = true ]; then
```
Comment (shrimalmadhur) [inline srs_setup.sh:21]: @antojoseph fyi - for hash verifications

File: .env.example [hunk 7]
```diff
@@ -45,8 +46,8 @@ NODE_ENABLE_NODE_API=true
 NODE_API_PORT=9091
 
 # goerli smart contracts
-NODE_EIGENDA_SERVICE_MANAGER=0x9FcE30E01a740660189bD8CbEaA48Abd36040010
-NODE_BLS_OPERATOR_STATE_RETRIVER=0x737Dd62816a9392e84Fa21C531aF77C00816A3a3
+NODE_EIGENDA_SERVICE_MANAGER=0xa3b1689Ab85409B15e07d2ED50A6EA9905074Ee5
```
Comment (shrimalmadhur) [inline .env.example:49]: @ian-shim need a second set of eyes to confirm these new ones are correct.

Comment (ian-shim) [reply .env.example:49]: Yes, that looks correct

**Pull request merged**